integrations/Refiner/browser.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable */
+import logger from "../../utils/logUtil";
+import { NAME } from "./constants";
+import { replaceUserTraits, replaceAccountTraits } from "./utils";
+
+class Refiner {
+  constructor(config) {
```

Comment:
Take a look at recently developed integrations and add below line:
```suggestion
  constructor(config, analytics) {
    if (analytics.logLevel) {
        logger.setLogLevel(analytics.logLevel);
      }
```